go/vt/worker/vertical_split_clone_cmd.go

```diff
@@ -121,10 +121,8 @@ func commandVerticalSplitClone(wi *Instance, wr *wrangler.Wrangler, subFlags *fl
 	if err != nil {
 		return nil, err
 	}
-	var tableArray []string
-	if *tables != "" {
-		tableArray = strings.Split(*tables, ",")
-	}
+	// the tableArray has zero elements when table flag is empty, which is an error case captured in function newVerticalSplitCloneWorker
```

Comment:
nit: Let's write comments as full sentences with proper punctuation.

That means:
- capitalize the first word
- add a period (".") at the end